src/component/DashboardTable/OrganizationGrid.tsx

```diff
@@ -0,0 +1,374 @@
+						<TableCell>Organization URL</TableCell>
+						<TableCell>Border Radius</TableCell>
+						<TableCell>Button Colors</TableCell>
+						<TableCell>Authentication Method</TableCell>
+						<TableCell align="center">Actions</TableCell>
+					</TableRow>
+				</TableHead>
+				<TableBody>
+					{paginatedRows.length === 0 ? (
+						<TableRow>
+							<TableCell colSpan={8}>
+								<Typography variant="body1">Organization not found.</Typography>
+							</TableCell>
+						</TableRow>
+					) : (
+						paginatedRows.map((row) => (
+							<TableRow key={row.id}>
+								<TableCell>{row.name}</TableCell>
+								<TableCell>{row.tagline}</TableCell>
+								<TableCell>{row.banner}</TableCell>
+								<TableCell>{row.organizationURL}</TableCell>
+								<TableCell>{row.borderRadius}</TableCell>
+								<TableCell>{row.buttonColors}</TableCell>
+								<TableCell>{row.authMethod}</TableCell>
+								<TableCell align="center">
+									<ButtonGroup variant="text" aria-label="text button group">
+										<Button>
+											<IconButton
+												aria-label={`edit-${row.id}`}
+												title="Edit"
+												sx={{
+													"&:hover": {
+														backgroundColor: "transparent",
+													},
+												}}
+												onClick={() => {
+													setSelectedRow(row);
+													setEditRowIndex(row.id);
+													setOpen(true);
+												}}
+											>
+												<Edit />
+											</IconButton>
+										</Button>
+										<Button>
+											<IconButton
+												aria-label={`delete-${row.id}`}
+												title="Delete"
+												onClick={() => handleRemoveOrg(row.id)}
+												sx={{
+													"&:hover": {
+														backgroundColor: "transparent",
+													},
+												}}
+											>
+												<Delete />
+											</IconButton>
+										</Button>
+									</ButtonGroup>
+								</TableCell>
+							</TableRow>
+						))
+					)}
+				</TableBody>
+			</Table>
+
+			<StyledPagination>
+				<Pagination
+					count={Math.ceil(filteredRows.length / itemsPerPage)}
+					variant="outlined"
+					page={page}
+					onChange={handlePageChange}
+				/>
+			</StyledPagination>
+
+			<StyledModal
+				open={open}
+				onClose={() => setOpen(false)}
+				aria-labelledby="modal-modal-title"
+				aria-describedby="modal-modal-description"
+			>
+				<Box
+					p={3}
+					borderRadius={3}
+					bgcolor={"background.default"}
+					color={"text.primary"}
+					component="form"
+					sx={{
+						display: "flex",
+						flexDirection: "column",
+						justifyContent: "space-between",
+						width: "auto",
+						maxWidth: "38vw",
+						height: "auto",
+						minHeight: "300px",
+						gap: "30px",
+					}}
+					noValidate
+					autoComplete="off"
+				>
+					<Box
+						sx={{
+							display: "flex",
+							justifyContent: "space-between",
+						}}
+					>
+						<Typography variant="h6" color="gray" textAlign="center">
+							Edit Organization
```

Comment:
The text "Edit Organization" should come from a language definition file. As well as other similar English texts